LayoutTests/imported/w3c/web-platform-tests/html/semantics/interactive-elements/the-dialog-element/dialog-focus-shadow-expected.txt

```diff
@@ -23,30 +20,24 @@ PASS show: Autofocus after, no delegatesFocus
-
-<!--
-  We focus this one between each test, to en...
+PASS show: Autofocus on shadow host, yes delegatesFocus, no siblings
+PASS showModal: Autofocus on shadow host, yes delegatesFocus, no siblings
 FAIL show: Autofocus on shadow host, yes delegatesFocus, sibling before assert_equals: expected Element node <div autofocus=""></div> but got Element node <button tabindex="-1">Focusable</button>
 FAIL showModal: Autofocus on shadow host, yes delegatesFocus, sibling before assert_equals: expected Element node <div autofocus=""></div> but got Element node <button tabindex="-1">Focusable</button>
-FAIL show: Autofocus on shadow host, yes delegatesFocus, sibling after assert_equals: expected Element node <div autofocus=""></div> but got Element node <button tabindex="-1">Focusable</button>
-FAIL showModal: Autofocus on shadow host, yes delegatesFocus, sibling after assert_equals: expected Element node <div autofocus=""></div> but got Element node <button tabindex="-1">Focusable</button>
+PASS show: Autofocus on shadow host, yes delegatesFocus, sibling after
+PASS showModal: Autofocus on shadow host, yes delegatesFocus, sibling after
 PASS show: Autofocus on shadow host, no delegatesFocus, no siblings
 PASS showModal: Autofocus on shadow host, no delegatesFocus, no siblings
 PASS show: Autofocus on shadow host, no delegatesFocus, sibling before
 PASS showModal: Autofocus on shadow host, no delegatesFocus, sibling before
 PASS show: Autofocus on shadow host, no delegatesFocus, sibling after
 PASS showModal: Autofocus on shadow host, no delegatesFocus, sibling after
-FAIL show: Autofocus inside shadow tree, yes delegatesFocus, no siblings assert_equals: expected Element node <div></div> but got Element node <button tabindex="-1" id="focus-between-tests">Focus betw...
-FAIL showModal: Autofocus inside shadow tree, yes delegatesFocus, no siblings assert_equals: expected Element node <div></div> but got Element node <body>
-
-<!--
-  We focus this one between each test, to en...
+PASS show: Autofocus inside shadow tree, yes delegatesFocus, no siblings
+PASS showModal: Autofocus inside shadow tree, yes delegatesFocus, no siblings
 PASS show: Autofocus inside shadow tree, yes delegatesFocus, sibling before
 PASS showModal: Autofocus inside shadow tree, yes delegatesFocus, sibling before
-PASS show: Autofocus inside shadow tree, yes delegatesFocus, sibling after
-PASS showModal: Autofocus inside shadow tree, yes delegatesFocus, sibling after
+FAIL show: Autofocus inside shadow tree, yes delegatesFocus, sibling after assert_equals: expected Element node <button tabindex="-1" class="focus-me">Focusable</button> but got Element node <div></div>
+FAIL showModal: Autofocus inside shadow tree, yes delegatesFocus, sibling after assert_equals: expected Element node <button tabindex="-1" class="focus-me">Focusable</button> but got Element node <div></div>
```

Comment:
What's up with these two failures?

Reply:
I'll followup with Domenic to figure out if this test needs fixing.